UWP/GalaxyExplorer/App.xaml.cs

```diff
@@ -125,6 +126,58 @@ private void InitializeUnity(string args)
 			}
 
 			Window.Current.Activate();
+			
+			// Integrate with Shell back button behavior on non-Holographic UWP platforms such as Desktop and Mobile
+			if (AnalyticsInfo.VersionInfo.DeviceFamily != "Windows.Holographic")
```

Comment:
I'm planning on staying with the DeviceFamily check (I couldn't access the UnityEngine check there and will need the SurfaceHub check in the future as well.)  Looks like I have a spaces v. tabs issue that came in a merge after updating the comment :(  I'll fix that in a future commit/pull request.